ICGE-Simulation/src/main/java/de/unistuttgart/informatik/fius/icge/simulation/actions/ActionLog.java

```diff
@@ -77,4 +77,18 @@
      *     The action to log
      */
     void logAction(Action actionToLog);
+    
+    /**
+     * Set ActionLog console output (default: true)
```

Comment:
```suggestion
     * Set ActionLog console output (default: {@code true})
```
Im param auch. (Kann auch `@Code` sein)